src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -309,25 +387,37 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
         throw Error(Errors.LedgerMultiAssetsNotSupported)
       }
     })
+
+    // TODO revisit, use a switch?
+    const usecase = determineUsecase(txAux.certificates, signingFiles)
+    if ((usecase === SignTxUsecases.SIGN_TX_USECASE_POOL_REGISTRATION_OPERATOR)
+      && !isFeatureSupportedForVersion(LedgerCryptoProviderFeature.POOL_REGISTRATION_OPERATOR)
+    ) {
+      throw Error(Errors.PoolRegistrationAsOperatorNotSupported)
+    }
   }
 
   const ledgerSignTx = async (
     txAux: _TxAux, signingFiles: HwSigningData[], network: Network, changeOutputFiles: HwSigningData[],
   ): Promise<LedgerWitness[]> => {
-    ensureFirmwareSupportsParams(txAux)
-    const { paymentSigningFiles, stakeSigningFiles } = filterSigningFiles(signingFiles)
```

Comment:
Is there any reason why was this removed? It's breaking old logic and causing wrong paths mapping.